oss_src/lambda/pylambda_worker.cpp

```diff
@@ -72,6 +72,8 @@ int EXPORT pylambda_worker_main(const std::string& root_path,
   LOG_DEBUG_WITH_PID("server_address = '" << server_address << "'");
   LOG_DEBUG_WITH_PID("parend pid = " << parent_pid);
 
+  size_t _last_line = __LINE__;
+#define __TRACK do { _last_line = __LINE__; } while(0)  
```

Comment:
This is the C way of wrapping macros. From what I understand there is a more straight forward way to do this in C++. You can introduce arbitrary nesting where ever you like. So I believe you can just do:

`#define __TRACK do { _last_line = __LINE__; }`